meta-balena-common/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -17,7 +18,8 @@ SRC_URI:append = " \
     "
 
 do_install:append() {
-    install -m 0755 ${WORKDIR}/prepare ${D}/init.d/70-prepare
+    install -m 0755 ${WORKDIR}/prepare ${D}/init.d/00-prepare
```

Comment:
Before we had:
```
000-console_null
00-debug
01-udev
70-prepare
```

I'd rather leave the prepare module after udev as it was before, but I am guessing prepare mounts somewhere where bootchar logs and we want bootchart to also include udev data, right?

I would prefer if the introduction of this bootchart module which is really only useful for OS development would not modify production code, like moving prepare before udev.

Reply:
@alexgg Correct, bootchart is intended to run as early as possible to profile the boot process, and udev (as well as many of the subprocesses it spawns) is one of the primary resource consumers during boot.

The prepare hook mounts the tmpfs at `/run`, which is where the bootchart output is written.

We could duplicate this in the bootchart hook in the `run()` function before running bootchart, but I found it cleaner to just mount `/run` earlier.

This should also result in more complete logs, especially when debugging udev rules.